pretrain/scripts/v3-converter/convert.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Model conversion script for converting Megatron format checkpoints into Huggingface format
+#
+# Usage:
+# On a cluster with SLURM:
+#   Run `sbatch --partition {partition} convert.sh SOURCE_DIR TARGET_DIR`
+# On a cluster without SLURM:
+#   Run `bash convert.sh SOURCE_DIR TARGET_DIR TEMPORAL_DIR > outpus/convert.out 2> outputs/convert.err`
+# - SOURCE_DIR: Megatron checkpoint directory including `iter_NNNNNNN`
+# - TARGET_DIR: Output directory for the Hugging Face format
+#
+# Example:
+# sbatch convert.sh /data/experiments/{exp-id}/checkpoints/iter_0001000 /data/experiments/{exp-id}/hf_checkpoints/iter_0001000 
+#
+# This script requires 1 node on the `gpu` partition on the cluster.
+
+#SBATCH --job-name=ckpt-convert
+#SBATCH --partition=<FIX_ME>
+#SBATCH --nodes=1
+#SBATCH --gpus=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=64
```

Comment:
SakuraのGPUノードはCoresPerSocket=56, CPUノードは60なので多すぎます。
そもそも64スレッドも回さないと思うので少量でよいと思います。

```suggestion
#SBATCH --cpus-per-task=8
```